games/game_stalkeranomaly.py

```diff
@@ -1,14 +1,23 @@
 # -*- encoding: utf-8 -*-
```

Comment:
That file is a bit too long and hard to navigate IMO. You should make a package `games.stalkeranomly` where you can have different files for the different classes (I'm not pro one-file per class but you can separate some of them at least), then you simply declare the game class in `game_stalkeranomaly.py`.